src/components/KuadrantPoliciesPage.tsx

```diff
@@ -212,6 +213,23 @@ const KuadrantPoliciesPage: React.FC = () => {
     <PoliciesListPage resource={resources[2]} activeNamespace={activeNamespace} />
   );
 
+  const handleNamespaceChange = (activeNamespace: string) => {
+    let basePath = '';
+    let activeTab = location.pathname.split('/').pop();
+    if (activeTab == 'policies') {
+      activeTab = '';
+    }
+
+    if (activeNamespace !== '#ALL_NS#') {
+      basePath = `/kuadrant/ns/${activeNamespace}/policies`;
+    } else {
+      basePath = `/kuadrant/all-namespaces/policies`;
+    }
+    const fullPath = `${basePath}/${activeTab}`;
+
+    history.replace(fullPath);
```

Comment:
I see what you're doing here, which makes sense.
Could it be simplified (more like it was before) by setting the variable in the if/else with the tab appended?
```suggestion
    if (activeNamespace !== '#ALL_NS#') {
      basePath = `/kuadrant/ns/${activeNamespace}/policies/${activeTab}`;
    } else {
      basePath = `/kuadrant/all-namespaces/policies/${activeTab}`;
    }

    history.replace(basePath);
```

`basePath` may not make sense as the variable name though.

Reply:
i added the more convoluted but to stop a unnecessary / appearing at the end of the policies it has not affect on the functionality so i can remove it to make the code more readble